### PR TITLE
Enable and configure Travis-CI.

### PR DESCRIPTION
**Pull request by jguiditta**
Signed-off-by: Jason Guiditta <jason.guiditta@gmail.com>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Config file for automatic testing at travis-ci.org
+sudo: false
+
+language: python
+python:
+  - 3.6
+  - 3.5
+
```
Comment (fuzzball81) [inline .travis.yml]: I would recommend getting python 3.7 testing added. https://github.com/release-depot/atkinson/pull/2 has an example that appears to work. Python 3.7 is probably low risk at this point, but I know other projects has seen issues between 3.6 and 3.7 with what appear to be ABI breaks.

File: tox.ini [hunk 2]
```diff
@@ -0,0 +1,20 @@
+[tox]
+envlist = py35, py36, flake8
+
+[travis]
+python =
+    3.6: py36
+    3.5: py35
+
```
Comment (fuzzball81) [inline tox.ini]: Same python 3.7 comment as above.

File: tests/test_rpmutils.py [hunk 3]
```diff
@@ -72,7 +72,7 @@ def test_labelCompare(self):
         f = ('0', '6.0.0', '0.2.0rc2.el7ost')
         g = ('0', '6.0.0', '0.3.0rc2.el7ost')
         # new dlrn
-        l = ('0', '6.1.1', '0.2.20170217063119.ad33b59.el7ost')
+        new_dlrn = ('0', '6.1.1', '0.2.20170217063119.ad33b59.el7ost')
```
Comment (fuzzball81) [inline tests/test_rpmutils.py]: Good cleanup here.

File: .travis.yml (hunk 1 above)
Comment (jguiditta) [reply .travis.yml]: I have no problem with that, but given that tests are passing right now, what would you think of merging this, and I will do that as my next patch?

Comment (fuzzball81) [reply .travis.yml]: +1 from me. I will look for a followup patch.

**Pull request merged**